examples/bri-3/components/storage/migrations/20220818160540-mydb.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  return null;
```

Comment:
dummy migration is not useful, just for illustration, but since we can create tables lets make proper migration
also there is migration introduced by @ognjenkurtic already i believe, let's do same approach/folders for it

Reply:
This file is also not needed. Migrations are handled through prisma commands for local development and will be triggered from the app on application startup in the future.